flink-core/src/main/java/org/apache/flink/configuration/description/HtmlFormatter.java

```diff
@@ -34,9 +36,22 @@ protected void formatLineBreak(StringBuilder state) {
 	}
 
 	@Override
-	protected void formatText(StringBuilder state, String format, String[] elements) {
+	protected void formatText(
+			StringBuilder state,
+			String format,
+			String[] elements,
+			EnumSet<TextElement.TextStyle> styles) {
 		String escapedFormat = escapeCharacters(format);
+
+		String prefix = "";
+		String suffix = "";
+		if (styles.contains(TextElement.TextStyle.CODE)) {
+			prefix = "<tt>";
```

Comment:
What I don't like is that `<tt>` or `<code>` might not get formatted the same way as markdown's backticks. Why don't we enable markdown syntax in HTML tags. There is a configuration option in Kramdown, the default HTML converter in Jekyll:

> By default, kramdown parses all block HTML tags and all XML tags as raw HTML blocks. However, this can be configured with the parse_block_html. If this is set to true, then syntax parsing in HTML blocks is globally enabled. It is also possible to enable/disable syntax parsing on a tag per tag basis using the markdown attribute:
>
>    If an HTML tag has an attribute markdown="0", then the tag is parsed as raw HTML block.
>
>    If an HTML tag has an attribute markdown="1", then the default mechanism for parsing syntax in this tag is used.
>
>    If an HTML tag has an attribute markdown="block", then the content of the tag is parsed as block level elements.
>
>    If an HTML tag has an attribute markdown="span", then the content of the tag is parsed as span level elements.

https://kramdown.gettalong.org/syntax.html